tests/temporal/test_temporal_codec.py

```
@@ -0,0 +1,67 @@
+    codec = EncryptionCodec(_KEY)
+    legacy = _payload(b'{"fax_uuid": "u"}')
+    (decoded,) = await codec.decode([legacy])
+    assert decoded == legacy
+
+
+@pytest.mark.asyncio
+async def test_wrong_key_fails_loudly_not_silently():
+    codec = EncryptionCodec(_KEY)
+    (encoded,) = await codec.encode([_payload(b"x")])
+    other = EncryptionCodec(Fernet.generate_key().decode())
+    with pytest.raises(Exception):
```

Ruff (0.16.3)

[warning] Do not assert blind exception: `Exception`

(B017)